client/src/features/catalog/ProductCard.tsx

```diff
@@ -9,9 +9,9 @@ interface Props {
 
 export default function ProductCard({ product }: Props) {
     return (
-        <div className="card" key={product.id}>
+        <div   key={product.id}>
         <Link to={`/product/${product.id}`}>
```

Comment:
code should be formatted properly - with proper indentations for nested elements.